data/policy_permissions_list/master_policy_permissions_list.yaml

```diff
@@ -1021,7 +1021,7 @@
       required: true
 - id: "./cost/aws/unused_ip_addresses/aws_unused_ip_addresses.pt"
   name: AWS Unused IP Addresses
-  version: '8.1'
+  version: 9.0.0
```

Comment:
I notice the version field values for other templates in this list are strings. Not sure why this one looks like a number

Can see it's the same for the Policy Template Synchronization version which follows the same notation `2.0.0`